### ISSUE TITLE
Add getting started section to README.md

### ISSUE THREAD

**Issue by Ryuno-Ki** [ionicc/Github-organization-inviter#16]
This is a follow-up of #8.

It took me a short moment to learn, how to get started with this project.
In the end https://github.com/ionicc/Github-organization-inviter/blob/5381d04484b94a64d836c6b215ce67827fa58e72/main.js#L1 gave me the final hint, that I had to call `node ./main.js` to start the express server.

Normally I'd expect to find a `npm start` in `package.json` (#15) to run the server.

**Comment by ionicc** [comment]
Yes, This goes with #15. The addition of package.json would fix this.
